packages/fcl/src/wallet-utils/send-msg-to-fcl.js

```diff
@@ -3,8 +3,10 @@ import {onMessageFromFCL} from "./on-message-from-fcl"
 export const sendMsgToFCL = (type, msg = {}) => {
   if (window.location !== window.parent.location) {
     window.parent.postMessage({...msg, type}, "*")
-  } else {
+  } else if (window.opener) {
     window.opener.postMessage({...msg, type}, "*")
```

Comment:
Might could throw here if opener no longer exists so wallet can handle a closed pop/tab

Reply:
If `window.opener` (parent) has been closed, it becomes null as well => this if statement will be false and it will move to the next else block where it throws.

Reply:
We already have error handling in place if popup has been closed and cannot be communicated with anymore (if that is what was meant)
https://github.com/onflow/fcl-js/blob/b605ca8b7f963261b3260413f8cc2fad91fe0635/packages/fcl/src/current-user/exec-service/strategies/utils/pop.js#L49